config/kubernetes/sig-docs/teams.yaml

```
@@ -231,13 +231,19 @@ teams:
     description: Approvers for Portuguese content
     members:
     - femrtnz
+    - jailton
     - jcjesus
+    - rikatz
+    - yagonobre
```

@jailton posso aproveitar o PR eu adicionar o meu user ai? não sei porque não estava :( 

```suggestion
    - yagonobre
    - jhonmike
```